### ISSUE TITLE
ImportError: cannot import name 'auto' from 'tqdm' 

### ISSUE THREAD

**Issue by andrewwwww12** [CiaraStrawberry/TemporalKit#72]
im getting this error: 
ImportError: cannot import name 'auto' from 'tqdm' (C:\ai\stable-diffusion-webui\venv\lib\site-packages\tqdm\__init__.py)

its happening both with the automatic1111 installation and the manual way. 

anyone got a fix? cant find anything anywhere

**Comment by alescdb** [comment]
I had the same issue :

1. Stop SD
2. Delete/rename `venv` directory
3. Restart SD

It should rebuild the `venv` and start correctly.

**Comment by AuberDjine** [comment]
> I had the same issue :
> 
> 1. Stop SD
> 2. Delete/rename `venv` directory
> 3. Restart SD
> 
> It should rebuild the `venv` and start correctly.

Nope, for me, as long as I dont also delete the TemporalKit extention the venv will just rebuild and then fail with the same error again.

**Comment by railgun233** [comment]
`source venv/bin/activate
pip install tqdm --upgrade
pip install moviepy --upgrade`

It is because the original moviepy version is too old. Update these packages can solve the problem.

**Comment by mighue** [comment]
same error: 
ImportError: cannot import name 'auto' from 'tqdm' (C:\SD-A1111\system\python\lib\site-packages\tqdm\__init__.py)

`source venv/bin/activate && pip install tqdm --upgrade && pip install moviepy --upgrade`  , tried this, but it still doesn't work

**Comment by Arta5500** [comment]
Delete the "tqdm" and "tqdm-4.65.0.dist-info" folders in "system\python\Lib\site-packages" and then launch SD. The deleted folders should be recreated. Hope this helps, it worked for me.

**Comment by usvendor** [comment]
> Delete the "tqdm" and "tqdm-4.65.0.dist-info" folders in "system\python\Lib\site-packages" and then launch SD. The deleted folders should be recreated. Hope this helps, it worked for me.

Tried that. Those folder do not get rebuild at all. Tried everything mentioned in this thread as well. Nothing works!

**Comment by yosshiiiiiiiiiii** [comment]
I was able to solve it as a programming amateur. I'll teach you how to do that, so I hope it helps someone.pip install --upgrade huggingface_hub.pip install --upgrade pip.Try updating the tqdm package to the latest version in the virtual environment (venv).Update moviepy to latest version in virtual environment.I was able to solve it by having CHATGPT read all the error sentences. Good luck

**Comment by swingppx** [comment]
I was having the same error, tried everything listed on different posts.
Turned out it was a problem with FFmpeg despite correctlyu having it enlisted in Path variables.

The following dumb way worked
Put ffprobe.exe to
SD root folder
SD root folder\system
SD root folder\webui

**Comment by weterynarzfred** [comment]
> source venv/bin/activate && pip install tqdm --upgrade && pip install moviepy --upgrade

This worked for me, except it was `source venv/Scripts/activate && pip install tqdm --upgrade && pip install moviepy --upgrade`
